app.py

```diff
@@ -1,36 +1,38 @@
-from os import environ
 from flask import Flask, render_template, url_for
 from os import environ
 from importlib import import_module
 from common import ServiceType
 
 app = Flask(__name__)
 
-def prepare_vars(vars):
-    missingvars = []
-    for var in vars:
-        env = environ.get(var)
-        if env == None:
-            missingvars.append(var)
+
+def prepare_vars(required_vars):
+    missing_vars = [var for var in required_vars if not environ.get(var)]
+    if missing_vars:
+        raise Exception(f"Please define variables: {', '.join(missing_vars)}")
+
+    for var in required_vars:
         globals()[var] = environ.get(var)
 
-    if(len(missingvars) > 0):
-        raise Exception("Please define variables: %s" % (", ".join(missingvars)) )
+
+prepare_vars(["PROJECT", "SITE_NAME", "HOST", "FLASK_ENV"])
```

Comment:
Like in https://github.com/samply/bridgehead/blob/4a1339540856eba47f4f97a274e74ee1990696ad/bridgehead#L72 The environment variable should be ENVIRONMENT